roles/openshift_metering/tasks/install.yaml

```diff
@@ -0,0 +1,99 @@
+---
+- name: Create temp directory for doing work in on target
+  command: mktemp -td openshift-metering-ansible-XXXXXX
+  register: mktemp
+  changed_when: False
+
+- name: Ensures {{ mktemp.stdout }}/metering-manifests dir exists
+  file:
+    path: "{{ mktemp.stdout }}/metering-manifests"
+    state: directory
+
+- name: Copy files to temp directory
+  copy:
+    src: "{{ item }}"
+    dest: "{{ mktemp.stdout }}/metering-manifests/{{ item | basename }}"
+  with_fileglob:
+  - "*.yaml"
+
+- name: Update Metering Helm Operator deployment image
+  yedit:
+    src: "{{ mktemp.stdout }}/metering-manifests/metering-helm-operator-deployment.yaml"
+    edits:
+    - key: spec.template.spec.containers[0].image
+      value: "{{ openshift_metering_operator_image }}"
+    - key: spec.template.spec.containers[1].image
+      value: "{{ openshift_metering_operator_image }}"
+  when: openshift_metering_operator_image != ""
+
+- name: Update Metering config
+  yedit:
+    src: "{{ mktemp.stdout }}/metering-manifests/metering.yaml"
+    edits:
+    - key: spec
+      value: "{{ openshift_metering_config }}"
+  when: openshift_metering_config != None
+
+- name: "Create {{ openshift_metering_namespace }} project"
+  oc_project:
+    state: present
+    name: "{{ openshift_metering_namespace }}"
+    description: Openshift Metering
+    node_selector: ""
+
+- name: Create metering namespace viewer cluster role
+  oc_clusterrole:
+    state: present
+    name: openshift-metering-namespace-viewer
+    rules:
+    - apiGroups: [""]
+      resources: ["namespaces"]
+      verbs: ["get"]
+
+- name: Check if namespace viewer clusterrolebinding exists
+  shell: >
+    {{ openshift_client_binary }} \
+      --config={{ openshift.common.config_base }}/master/admin.kubeconfig \
+      --namespace {{ openshift_metering_namespace }} \
+      get clusterrolebinding \
+      openshift-metering-namespace-viewer-{{ openshift_metering_namespace }} \
+      --ignore-not-found
+  register: rolebinding_exists
+
+- name: Grant access to openshift-metering with namespace viewer role
+  when: rolebinding_exists.stdout == ""
+  shell: >
+    {{ openshift_client_binary }} \
+      --config={{ openshift.common.config_base }}/master/admin.kubeconfig \
+      --namespace {{ openshift_metering_namespace }} \
+      create clusterrolebinding \
+      openshift-metering-namespace-viewer-{{ openshift_metering_namespace }} \
+      --clusterrole \
+      openshift-metering-namespace-viewer \
+      --serviceaccount \
+      "{{ openshift_metering_namespace }}:metering" || true
```

Comment:
do we still need `|| true` since we have the when condition now?

Reply:
If not, can we also change this to be `command` instead of `shell`?

Reply:
Oh right, I forgot to update this part.